### DISCUSSION TITLE
outgoing links: skip "are you sure you want to leave"  #safetyism-sucks

### DISCUSSION THREAD

**Discussion by allanlaal** [code-for-charity/Wish-Factory#15] [Proposals & Voting, Ranking]
SHORT Table | (Summary)     
------------ | -------------   
*Problem*     |                                   
*Solution*     |     https://greasyfork.org/en/scripts/439458-youtube-outgoing-links-fix/code      
*Alternatives*|         
*Scope*         |           
*Side effects*|        
*Context*      |

**Comment by pickanothername** [comment]
I appreciate the greasyfork link, makes my job much easier 😂

**Comment by pickanothername** [comment]
@allanlaal Can you please explain how to reproduce this?
I'm not getting a "are you sure you want to leave" message upon clicking external links

**Comment by DandelionSprout** [comment]
The developer of *I Don't Care About Cookies*, did get a GitHub account sometime after the cited thread: https://github.com/kiboke
